tests/pre-command.bats

```diff
@@ -46,6 +46,20 @@ load '/usr/local/lib/bats/load.bash'
   unstub aws
 }
 
+@test "Uses endpoint URL" {
+  export BUILDKITE_PLUGIN_AWS_S3_SYNC_SOURCE=s3://source
+  export BUILDKITE_PLUGIN_AWS_S3_SYNC_DESTINATION=destination/
+  export BUILDKITE_PLUGIN_AWS_S3_SYNC_ENDPOINT_URL=envato-test-bucket.s3-website-us-east-1.amazonaws.com
```

Comment:
In the case of AWS, would the `s3://<bucket>` location be different from the endpoint url?

Reply:
With aws I do not believe you would use this option (unless you are testing against a local s3 server or integrating with a third party compatible service)